src/components/Synthetics/SelectorBase/SelectorBase.tsx

```diff
@@ -81,53 +82,66 @@ export function SelectorBaseMobileButton(
 }
 
 export function SelectorBaseDesktopRow(
-  props: React.HTMLAttributes<HTMLTableRowElement> & {
+  props: PropsWithChildren<{
     disabled?: boolean;
     disabledMessage?: ReactNode;
     message?: ReactNode;
-  }
+    className?: string;
+    onClick?: (e: React.MouseEvent) => void;
+  }>
 ) {
-  const body = useMemo(() => {
-    if (props.message) {
-      return <TooltipWithPortal content={props.message}>{props.children}</TooltipWithPortal>;
-    }
-
-    return props.children;
-  }, [props.children, props.message]);
-
   if (props.disabled && props.disabledMessage) {
     return (
       <TooltipWithPortal
-        as="tr"
-        className={cx("SelectorBaseUtils-row", props.className)}
+        as={TableTr}
+        className={cx("SelectorBaseUtils-row ", props.className)}
```

Comment:
Unneeded space

Reply:
removed, also removed the same issue sevelar lines below